pkg/testing/integration/program.go

```diff
@@ -931,8 +921,8 @@ func (pt *ProgramTester) runPipenvCommand(name string, args []string, wd string)
 		}
 
 		if pt.opts.Verbose {
-			fprintf(pt.opts.Stdout, "acquired pip install lock\n")
-			defer fprintf(pt.opts.Stdout, "released pip install lock\n")
+			pt.t.Log("acquired pip install lock")
+			defer pt.t.Log("released pip install lock")
```

Comment:
Just checking, we're dropping the newlines because `t.Log` already includes a newline?

Reply:
Correct - logf will handle the newlines